text-ui-test/ACTUAL.TXT

```diff
@@ -0,0 +1,72 @@
+Greetings mortal, my name is
+ ____        _        
+|  _ \ _   _| | _____ 
+| | | | | | | |/ / _ \
+| |_| | |_| |   <  __/
+|____/ \__,_|_|\_\___|
+
+How may I help you?
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+	ADDED : [D][ ]  revise for 2103T  (due by:  today, 12pm)
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+	ADDED : [E][ ]  school dance (at:  2nd Feb 2020)
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+	ADDED : [T][ ]  clean room
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+	ADDED : bruhhh
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+	Here is your list:
+	(1) [D][ ]  revise for 2103T  (due by:  today, 12pm)
+	(2) [E][ ]  school dance (at:  2nd Feb 2020)
+	(3) [T][ ]  clean room
+	(4) [ ] bruhhh
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+	I've marked (4) as done!
+	(4) [X] bruhhh
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+	You have entered an invalid index.
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+	Here is your list:
+	(1) [D][ ]  revise for 2103T  (due by:  today, 12pm)
+	(2) [E][ ]  school dance (at:  2nd Feb 2020)
+	(3) [T][ ]  clean room
+	(4) [X] bruhhh
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
+	See u the next time you test ur code :P
+
+=^..^=   =^..^=   =^..^=    =^..^=    =^..^=    =^..^=    =^..^=
+
```

Comment:
Looks much better than mine T.T

Reply:
I think I spent a stupid amount of time deciding on how to personalize my bot HAHA. Anyways, thanks!